test/Installer/Microsoft.DotNet.Cli.Msi.Tests/Microsoft.DotNet.Cli.Msi.Tests.csproj

```diff
@@ -25,4 +25,8 @@
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
nit: revert (VS  bug adds this sometimes)

Reply:
done, thanks. btw, that's a bug? And I just updated to latest, seems it is still there

Reply:
well, i consider it a bug.